website/docs/r/search_service.html.markdown

```diff
@@ -8,79 +8,83 @@ description: |-
 
-* `resource_group_name` - (Required) The name of the resource group in which to create the Search Service. Changing this forces a new resource to be created.
+* `resource_group_name` - (Required) The name of the Resource Group where the Search Service should exist. Changing this forces a new Search Service to be created.
 
-* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+* `sku` - (Required) The SKU which should be used for this Search Service. Possible values are `basic`, `free`, `standard`, `standard2` and `standard3` Changing this forces a new Search Service to be created.
 
-* `sku` - (Required) Valid values are `basic`, `free` and `standard`. `standard2` and `standard3` are also valid, but can only be used when it's enabled on the backend by Microsoft support. `free` provisions the service in shared clusters. `standard` provisions the service in dedicated clusters.  Changing this forces a new resource to be created.
+-> The `basic` and `free` SKU's provision the Search Service in a Shared Cluster - the `standard` SKU's use a Dedicated Cluster.
 
-* `replica_count` - (Optional) Default is 1. Valid values include 1 through 12. Valid only when `sku` is `standard`. Changing this forces a new resource to be created.
+~> **Note:** The SKU's `standard2` and `standard3` are only available when enabled on the backend by Microsoft.
 
-* `partition_count` - (Optional) Default is 1. Valid values include 1, 2, 3, 4, 6, or 12. Valid only when `sku` is `standard`. Changing this forces a new resource to be created.
+---
 
-* `tags` - (Optional) A mapping of tags to assign to the resource.
+* `partition_count` - (Optional) The number of partitions which should be created. Possible values are `1`, `2`, `3`, `4`, `6`, or `12`. Defaults to `1`. Changing this forces a new Search Service to be created.
 
-## Attributes Reference
+* `replica_count` - (Optional) The number of replica's which should be created. Changing this forces a new Search Service to be created.
 
-The following attributes are exported:
+-> **Note:** `partition_count` and `replica_count` can only be configured when using a `standard` sku.
 
-* `id` - The Search Service ID.
+* `tags` - (Optional) A mapping of tags which should be assigned to the Search Service.
 
-* `primary_key` - The Search Service Administration primary key.
+## Attributes Reference
 
-* `secondary_key` - The Search Service Administration secondary key.
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the Search Service.
+
+* `primary_key` - The Primary Key used for Search Service Administration.
 
 * `query_keys` - A `query_keys` block as defined below.
 
+* `secondary_key` - The Secondary Key used for Search Service Administration.
+
 ---
 
 A `query_keys` block exports the following:
 
-* `name` - The name of the query key.
+* `key` - The value of this Query Key.
```

Comment:
Same here, alphabetical vs typical

Reply:
in this instance these should be alphabetical for computed blocks, so I think this is fine?